Gopkg.lock

```diff
@@ -815,6 +815,14 @@
   revision = "6b638e95a32d0c1131db0e7fe83775cbea4a0d0b"
   version = "v0.1.0"
 
+[[projects]]
+  branch = "master"
+  digest = "1:1c96351a94bb68711cd7790e64cffa35d11da796c052aca1fe2b83bda856ef44"
+  name = "github.com/naoina/toml"
```

Comment:
I don't think this package should be included, perhaps missed an import line in github.com/influxdata/toml

Reply:
:no_mouth: oops, I sure did